python/ray/log_monitor.py

```diff
@@ -96,11 +97,10 @@ def update_log_filenames(self):
             if full_path not in self.log_filenames:
                 self.log_filenames.add(full_path)
                 self.closed_file_infos.append(
-                    LogFileInfo(
-                        filename=full_path,
-                        size_when_last_opened=0,
-                        file_position=0,
-                        file_handle=None))
+                    LogFileInfo(filename=full_path,
```

Comment:
There are a bunch of formatting changes in this PR. I think you probably have a newer version of `yapf` installed. Can you install `yapf==0.23.0` and rerun the formatting? This is the version of yapf we use in Travis.